LPFilter/Source/PluginProcessor.cpp

```diff
@@ -163,16 +175,25 @@ void LpfilterAudioProcessor::processBlock (AudioSampleBuffer& ioBuffer, MidiBuff
     for (int i = totalNumInputChannels; i < totalNumOutputChannels; ++i)
         ioBuffer.clear (i, 0, ioBuffer.getNumSamples());
     
+    
     //Update frequency parameter
     updateParameters();
-    
-    // Filtering with Juce Modules
-    //juceModulesProcess (ioBuffer);
-    
-    // Filtering with DSPFilters
-    dspFiltersProcess (ioBuffer);
-    
-    // Filtering with custom filter
+
+    if (mode->getIndex() == 0)
```

Comment:
During a unit test, if the mode was not valid then the customProcessing would take place.
In general, this `if-else` statement is not wrong but it would be better to do the following
mode = 0 --> juce processing
mode = 1 --> dsp processing
mode = 2 --> custom processing
else --> cause an assertion (see Juce `jassertfalse`) and do not process the ioBuffer